src/views/repository/Home.vue

```diff
@@ -32,32 +34,38 @@
           <p v-if="repository.description">{{ repository.description }}</p>
           <p v-else>There is no description for this repository</p>
         </div>
-        <h1 class="bh-title-1">Can identify</h1>
-        <div class="bh-grid">
-          <div class="repository-home__info__attribute bh-grid__item">
-            <h1 class="bh-title-2">Intenties</h1>
-            <div class="repository-home__info__attribute__card">
-              <bh-badge
-                v-for="(intent, index) in repository.intents"
-                :key="index"
-                size="small"
-                color="grey"
-                class="repository-home__info__attribute__card__badge">
-                <span v-if="intent">{{ intent.value }}</span>
-              </bh-badge>
+        <div v-if="hasIntents || hasLabels">
+          <h1 class="bh-title-1">Can identify</h1>
+          <div class="bh-grid">
+            <div
+              v-if="hasIntents"
+              class="repository-home__info__attribute bh-grid__item">
+              <h1 class="bh-title-2">Intenties</h1>
+              <div class="repository-home__info__attribute__card">
+                <bh-badge
+                  v-for="(intent, index) in repository.intents"
+                  :key="index"
+                  size="small"
+                  color="grey"
+                  class="repository-home__info__attribute__card__badge">
+                  <span v-if="intent">{{ intent.value }}</span>
```

Comment:
Why you check if intent exists?

Reply:
because before there was no 'if conditional' to check if the list was not empty